BackofficeBundle/Resources/views/BackOffice/Underscore/flashBag._tpl.twig

```diff
@@ -1,3 +1,13 @@
 <div class="flashbag">
-    <%= html %>
+    <% if (typeof errors.error != 'undefined') { %>
```

Comment:
You can't change this tpl without changing widget/statusView.coffee which use it too.